osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -129,10 +129,10 @@ protected override void OnUserChange(T value)
 
         private void playSample(T value)
         {
-            if (Clock == null || Clock.CurrentTime - lastSampleTime <= 50)
```

Comment:
were these changes also automated? could you tell us which plugin / refactoring tool you used to perform these?